src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;

```

Comment:
The name of 'firstSide' variable is confusing, it would be better to name it 'side' because a square has all sides equal.